delta-lake/common/src/main/databricks/scala/com/nvidia/spark/rapids/delta/RapidsDeltaSQLConf.scala

```diff
@@ -67,6 +67,13 @@ trait RapidsDeltaSQLConf {
       .doc("Maximum amount of data for auto compaction.")
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("20GB")
+
```

Comment:
Update copyrights

Reply:
We have [pre commit hooks](https://github.com/NVIDIA/spark-rapids/blob/branch-24.06/CONTRIBUTING.md#pre-commit-hooks) to update copyrights automatically.